src/main/java/io/apimatic/core/logger/LoggerUtilities.java

```diff
@@ -31,37 +32,37 @@ private LoggerUtilities() {
     }
 
     /**
-     * Retrieves the headers to be logged based on the provided logging options,
+     * Retrieves the headers to be logged based on the provided logging configuration,
      * headers, and sensitivity masking configuration.
-     * @param loggingOptions       The logging options containing configurations for
+     * @param loggingConfiguration The logging configuration containing configurations for
      *                             header logging.
      * @param headers              The headers to be evaluated for logging.
      * @param maskSensitiveHeaders Determines whether sensitive headers should be
      *                             masked in the log.
      * @return A map containing the headers to be logged, considering the provided
-     *         options and sensitivity masking.
+     * configuration and sensitivity masking.
      */
-    public static Map<String, String> getHeadersToLog(LoggingOptions loggingOptions,
-            Map<String, String> headers, boolean maskSensitiveHeaders) {
+    public static Map<String, String> getHeadersToLog(HttpLoggingConfiguration loggingConfiguration,
+                                                      Map<String, String> headers,
+                                                      boolean maskSensitiveHeaders) {
         Map<String, String> extractedHeaders = LoggerUtilities.extractHeadersToLog(headers,
-                loggingOptions.getHeadersToInclude(), loggingOptions.getHeadersToExclude());
+                loggingConfiguration.getHeadersToInclude(),
+                loggingConfiguration.getHeadersToExclude());
 
-        Map<String, String> headersToLog = LoggerUtilities.filterSensitiveHeaders(extractedHeaders,
-                loggingOptions.getHeadersToUnmask(), maskSensitiveHeaders);
-
-        return headersToLog;
+        return LoggerUtilities.filterSensitiveHeaders(extractedHeaders,
+                loggingConfiguration.getHeadersToUnmask(), maskSensitiveHeaders);
     }
 
     /**
      * Extracts headers to log based on inclusion and exclusion criteria.
-     *
      * @param headers          The map of headers.
      * @param headersToInclude The set of headers to include.
      * @param headersToExclude The set of headers to exclude.
      * @return The extracted headers to log.
      */
     public static Map<String, String> extractHeadersToLog(Map<String, String> headers,
-            List<String> headersToInclude, List<String> headersToExclude) {
+                                                          List<String> headersToInclude,
```

Comment:
I don't see this convention in Java normally but is this a recommendation from the linter or you just read about it somewhere?

Reply:
This might be due to my local code formatter settings, I have reverted the changes.